### ISSUE TITLE
CPU warning message spams logs

### ISSUE THREAD

**Issue by schontz** [aws/amazon-chime-sdk-js#2241]
## What happened?

We are getting logs spammed with `[WARN] CPU warning: cpu`, posting many times per second in some cases. This is counter-productive because it is typical that we do things with logs, i.e., queue them and upload, all things which will take more CPU if logs are getting spammed.

## What did you expect to happen?

I expect that we would get this warning once when CPU spikes (assuming that that's what it means), and not get logged again until it recovers and spikes again.

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

1. Join a meeting on a machine with CPU constraints (VMs are a good one for repro)
2. Enable Voice Focus
3. Watch your logs fill up!

### Amazon Chime SDK for JavaScript version

3.1.0

### What browsers are you seeing the problem on?

Chrome Linux

### Browser version

101

### Browser console logs

```
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
[WARN] CPU warning: cpu
```

**Comment by schontz** [comment]
Yes, that seems appropriate. I would suggest a quick backoff. Also, do it via a timestamp (e.g., `Date.now()`) versus tracking with a timeout.

**Comment by richhx** [comment]
Yep that's exactly what I was thinking. I can look into making the change.

**Comment by richhx** [comment]
Changes have been merged and should be in release 3.4.0. Let me know if you still have issues or other questions.
